core/GlobalState.cc

```diff
@@ -48,7 +48,9 @@ const int Symbols::MAX_PROC_ARITY;
 
 GlobalState::GlobalState(shared_ptr<ErrorQueue> errorQueue)
     : globalStateId(globalStateIdCounter.fetch_add(1)), errorQueue(std::move(errorQueue)),
-      lspQuery(lsp::Query::noQuery()) {
+      lspQuery(lsp::Query::noQuery()), epochMutex(make_shared<absl::Mutex>()),
```

Comment:
Why an absl::Mutex vs std::mutex ooc?

Reply:
We use abseil mutexes in a bunch of places already, and it comes with many useful features.

https://abseil.io/docs/cpp/guides/synchronization